core/src/main/java/org/elasticsearch/search/highlight/HighlightBuilder.java

```diff
@@ -21,18 +21,12 @@
 
 import org.apache.lucene.search.Query;
 import org.apache.lucene.search.vectorhighlight.SimpleBoundaryScanner;
-import org.elasticsearch.common.ParsingException;
-import org.elasticsearch.common.io.stream.StreamInput;
```

Comment:
These are just a leftover but it feels silly to go add unused imports back to the file just to remove it from the diff.